cobalt/dom/media_source.cc

```diff
@@ -497,7 +511,7 @@ scoped_refptr<TimeRanges> MediaSource::GetSeekable() const {
   }
 
   if (source_duration == std::numeric_limits<double>::infinity()) {
-    scoped_refptr<TimeRanges> buffered = attached_element_->buffered();
+    scoped_refptr<TimeRanges> buffered = GetBufferedRange();
```

Comment:
Have we compared the value of `GetBufferedRange()` and `attached_element_->buffered()` here, to see if they are the same?